File: Overture.Web/clientapp/src/app/_models/review.ts
export class Review {
  id:string;
  businessId:string;
  businessName:string;
  reviewer:string;
  reviewerName:string;
  reviewDate:Date;
  content:string;
  rating:number;
  satisfied:boolean;
  recommend:boolean;
  onTime:boolean;
  onBudget:boolean;
}

File: Overture.Core/Domain/Entities/Review.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Overture.Core.Domain.Entities
{
	public class Review : IEntity
	{
		public Guid Id { get; set; }
		public Guid BusinessId { get; set; }
		public string Reviewer { get; set; } 
		public DateTime ReviewDate { get; set; }
		public string Content { get; set; }
		public int Rating { get; set; }
		public bool Satisfied { get; set; }
		public bool Recommend { get; set; }
		public bool Rehire { get; set; }
		public bool OnTime { get; set; }
		public bool OnBudget { get; set; }
    }
}
